bootstrap/pkg/apis/apps/configconverters/converters.go

```diff
@@ -49,7 +49,7 @@ func LoadConfigFromURI(configFile string) (*kfconfig.KfConfig, error) {
 	//
 	// TODO(jlewi): Should we use hashicorp go-getter.GetAny here? We use that to download
 	// the tarballs for the repos. Maybe we should use that here as well to be consistent.
-	appFile := path.Join(appDir, KfConfigFile)
+	appFile := path.Join(appDir, "tmp_app.yaml")
```

Comment:
Doesn't this function need to set kfconfig.KfConfig.ConfigFile? So that it can be referenced later.

LoadConfigFromURI is called from coordinator.go 
https://github.com/kubeflow/kubeflow/blob/7727e577413a63cb4134258107be312c3b72fe04/bootstrap/pkg/kfapp/coordinator/coordinator.go#L277

I think LoadConfigFromURI should set the URI from which the kfdefconfig was loaded.

LoadKFAppCfgFile calls CreateKfAppFile to actually persist the file
https://github.com/kubeflow/kubeflow/blob/7727e577413a63cb4134258107be312c3b72fe04/bootstrap/pkg/kfapp/coordinator/coordinator.go#L285

That eventually calls WriteConfigToFile

I think WriteConfigToFile could transform the URI to a local path.

Reply:
I am doing it in LoadConfigFromURI now.